game-core/src/main/java/games/strategy/net/ServerMessenger.java

```diff
@@ -478,7 +478,8 @@ private boolean isNameTaken(final String nodeName) {
         .anyMatch(nodeName::equalsIgnoreCase);
   }
 
-  public String getUniqueName(String currentName) {
+  public String getUniqueName(final String initialCurrentName) {
+    String currentName = initialCurrentName;
```

Comment:
initialCurrentName? 
Why not call it initialName instead?

Reply:
Yeah, I was pretty much just following a pattern with a lot of these changes.  You bring up a good point about this parameter, in particular.  After looking at it again, even just `name` might be suitable.  I'll push another commit soon.

Reply:
534d454e3